ts-orm/README.md

```diff
@@ -5,9 +5,13 @@
 ## Compile and Deploy with Klotho
 
 run the terminal commands:
+```sh
+npm i && npx tsc
+```
+
 ```sh
 # Compile the app
-klotho ./dist/index.js --app sample-ts-orm
+klotho . --app sample-ts-orm
```

Comment:
Have we switched from using `dist` to using `.`? (Or, were we always using `.` and I didn't notice?)